### DISCUSSION TITLE
Release v1.5.1 / 5.60.1

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2398] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes various issues and minor improvements.

**Note:** if you encounter the new message **SBIE2227**, the right course of action is to enable 8.3 name support for the volume you are using. Assuming you are using for example a RAM disk with the letter `R:\`, use the following command in an elevated command prompt:

```
fsutil behavior set disable8dot3 R: 0
```

This is not a new issue, but an old forgotten one. 
The message has been added to bring attention to potential problems and offer instructions how to properly resolve the issue.

#### Known Issues
- the legacy 32-bit build does not work properly #2401 - use the last 1.4.2 build instead: https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.4.2

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- added SBIE2227 warning when a sandbox is located on a volume without 8.3 name support
- added template for Tencent QQ [#2367](https://github.com/sandboxie-plus/Sandboxie/issues/2367)

### Fixed
- fixed issues when renaming a sandbox with a custom path [#2368](https://github.com/sandboxie-plus/Sandboxie/issues/2368)
- properly fixed the Firefox 106 issue [46e9979](https://github.com/sandboxie-plus/Sandboxie/commit/46e99799e2b753b85224dc921ec141a05025acfd)
- fixed issue with alternative UI modes [#2380](https://github.com/sandboxie-plus/Sandboxie/issues/2380)
- fixed command line corruption with breakout processes [#2377](https://github.com/sandboxie-plus/Sandboxie/issues/2377)
- fixed issues with Privacy Enhanced box types [#2342](https://github.com/sandboxie-plus/Sandboxie/issues/2342)
- fixed issue with boxed object directory initialization [#2342](https://github.com/sandboxie-plus/Sandboxie/issues/2342)
- Sandboxie no longer leaves behind permanent directory objects
- FIXED SECURITY ISSUE ID-21 AlpcConnectPortEx was not filtered by the driver [#2396](https://github.com/sandboxie-plus/Sandboxie/issues/2396)
- fixed issues with program control options [#2395](https://github.com/sandboxie-plus/Sandboxie/issues/2395)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.5.1'>Release v1.5.1 / 5.60.1</a>.</em>

**Comment by bastik-1001** [comment]
> FIXED SECURITY ISSUE ID-21 AlpcConnectPortEx was not filtered by the driver

Since when was this the case? E.g. pretty much since ever or because of changes since it got Open Source.

**Reply by DavidXanatos** [reply]
Like all other security issues in sandboxie, this one also was present since always. MSFT added AlpcConnectPortEx with windows 8 and no one thought about filtering it like AlpcConnectPort was filtered.

**Comment by topazcoding** [comment]
Hi David, very odd issue here. using any keyboard the letter p is blocked.

any idea ?

**Reply by isaak654** [reply]
Cannot reproduce, please provide step-by-step details.

**Comment by topazcoding** [comment]
This only happens on 1 of my machines, the letter p is not working on my keyboard when running anything sandboxed.

I have chrome opened not sandoboxed the letter P is working, as soon as I run chrome (or any app) sanboxed P simply stops working.

freaky.

**Reply by adamhj** [reply]
Got exactly the same problem minutes age, and fixed it by press win+p. I guess that the windows key was stuck in sandboxie envrionment
